lib/ebay_api/operations/sell/inventory/offers/get_listing_fees.rb

```diff
@@ -0,0 +1,21 @@
+class EbayAPI
+  scope :sell do
+    scope :inventory do
+      scope :offers do
+        # https://developer.ebay.com/api-docs/sell/inventory/resources/offer/methods/getListingFees
+        operation :get_listing_fees do
+          option :offer_ids,
+                 ->(ids) { Array(ids).map { |id| { offerId: id.to_s } } },
```

Comment:
Not sure that it is the best place for data transformation. Maybe it's better to move it to the ebay_api's model. @nepalez, WDYT?

Reply:
I reformatted the code here to fit it into the 80 characters limit but did not modify it otherwise. BTW I'm not using ebay_api models (yet).